pytoda/smiles/smiles_language.py

```diff
@@ -300,3 +494,250 @@ def smiles_to_selfies(self, smiles: str) -> str:
                 'the SMILES instead'
             )
             return smiles
+
+
+class SELFIESLanguage(SMILESLanguage):
+    """
+    SELFIESLanguage is a SMILESLanguage with a different default tokenizer.
+
+    SMILESLanguage handle SMILES data defining the vocabulary and
+    utilities to manipulate it, including encoding to token indexes.
+    """
+
+    def __init__(
+        self,
+        name: str = 'selfies-language',
+        smiles_tokenizer: SMILESTokenizer = tokenize_selfies,
+    ) -> None:
+        super().__init__(name=name, smiles_tokenizer=smiles_tokenizer)
+
+
+class SMILESEncoder(SMILESLanguage):
```

Comment:
That is true, but working on the class it was important to have the base class close by. I think this is also beneficial for anybody looking at the code.
I don't feel strongly about this, if you'd like me to I'll move it.

Reply:
I do somehow care about the import statements. `from pytoda.smiles.smiles_language import SMILESEncoder` is not very intuitive, but this could be fixed in the `__init__.py`